ops/prod_dispatch.yaml

```diff
@@ -0,0 +1,60 @@
+# This dispatch file applies to prod, not local dev
+# CI will renme this file appropriately and deploy it
+# The "default" module means "py2"
+# Cleaning this up is tracked in https://github.com/the-blue-alliance/the-blue-alliance/issues/5388
+
+dispatch:
+# Pin these pages to py2
+- url: "www.thebluealliance.com/_/account/*"
+  service: default
+- url: "www.thebluealliance.com/search*"
+  service: default
+# match py2 css and javascript (py3 is prefixed with py3_)
+- url: "www.thebluealliance.com/css/*"
+  service: default
+- url: "www.thebluealliance.com/js/*"
+  service: default
+  
+# APIs on py3 (both apiv3 and trusted api)
+- url: "www.thebluealliance.com/api/*"
+  service: py3-api
+
+# Send low-frequency long-running tasks to backend module
+# Uses B2 instance for higher CPU/memory limits
+- url: "*/backend-tasks-b2/*"
+  service: py3-tasks-cpu
+
+# Send low-frequency long-running tasks to backend module
+- url: "*/backend-tasks/*"
+  service: py3-tasks-io
+
+# Handles latency-insensive tasks
+- url: "*/tasks/*"
+  service: py3-tasks-io
+
+# API docs (swagger)
+- url: "*/swagger/*"
+  service: py3-web
+
+# mobile client API to py3
+- url: "*/clientapi/*"
+  service: py3-api
```

Comment:
This change is inherited from https://github.com/the-blue-alliance/the-blue-alliance/commit/3dafb6697bd9d5511afaf7240eab733bd11b26a6